tasks/run.py

```diff
@@ -0,0 +1,132 @@
+from invoke import task
+from multiprocessing import cpu_count
+from subprocess import run, PIPE, STDOUT
+from os.path import join, exists
+from tasks.util import RESULTS_DIR, COVID_DIR, BUILD_DIR
+import os
+import re
+
+COVID_SIM_EXE = join(BUILD_DIR, "src", "CovidSim")
+DATA_DIR = join(COVID_DIR, "data")
+
+IMAGE_NAME = "experiment-covid"
+RESULTS_FILE = join(RESULTS_DIR, "covid_native.csv")
+
+DEFAULT_COUNTRY = "Malta"
+NUM_CORES = cpu_count()
+NUM_REPEATS = 3
+
+# -----------------------------------
+# Constants from the original source
+UNITED_STATES = ["United_States"]
+CANADA = ["Canada"]
+
+USA_TERRITORIES = [
+    "Alaska",
+    "Hawaii",
+    "Guam",
+    "Virgin_Islands_US",
+    "Puerto_Rico",
+    "American_Samoa",
+]
+
+NIGERIA = ["Nigeria"]
+# -----------------------------------
+
+
+@task
+def native(local=True, country=DEFAULT_COUNTRY):
+    """
+    Runs the native experiment
+    """
+    if not local:
+        print("Remote not yet implemented")
+        exit(1)
+
+    # Write header to results
+    with open(RESULTS_FILE, "w") as out_file:
+        out_file.write("Country,Threads,Run,Time(s)\n")
```

Comment:
Writing results directly to a simple CSV makes them more readable and work very nicely with Python tools like Pandas